tests/test_yaml.py

```diff
@@ -61,13 +64,19 @@ def test_invalid_line():
 
     assert config.as_text() == """line_a: True
 line_b: !2
-        ^
---------
-Basic YAML parsing error
-- could not determine a constructor for the tag '!2'
---------
+#       ^
+# --------
+# Basic YAML parsing error
+# - could not determine a constructor for the tag '!2'
+# --------
 line_c: 8"""
 
+    # We want to make sure we're getting unicode back
+    if sys.version_info.major < 3:
+        assert isinstance(config.as_text(), unicode)
```

Comment:
@davegaeddert I'm not seeing the error you're mentioning, but the pypy builds seem to have hung at this point.